projects/ngrx.io/content/examples/testing-store/stackblitz.json

```diff
@@ -0,0 +1,4 @@
+{
+  "description": "Store Testing Tutorial",
+  "files": ["!**/*.d.ts", "!**/*.js"]
```

Comment:
The `files` array needs to include a glob for the `spec` files so they get included in the generated StackBlitz.